src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,67 @@
+.clock {
+  display: flex;
+  position: relative;
```

Comment:
1. Perhaps need to add height and weight with ur $cloak-size. It will become the hover area.